### PR TITLE
Create entities and relations for Hotel, Image, Room and Address

### PR DESCRIPTION
**Pull request by tringuyennfq**
Database Diagram: https://dbdiagram.io/d/62b96a4569be0b672c4927c6

### CODE REVIEW

File: src/Repository/AddressRepository.php [hunk 1]
```diff
@@ -0,0 +1,66 @@
...
+
+/**
+ * @extends ServiceEntityRepository<Address>
+ *
+ * @method Address|null find($id, $lockMode = null, $lockVersion = null)
+ * @method Address|null findOneBy(array $criteria, array $orderBy = null)
+ * @method Address[]    findAll()
+ * @method Address[]    findBy(array $criteria, array $orderBy = null, $limit = null, $offset = null)
+ */
+class AddressRepository extends ServiceEntityRepository
```
Comment (khoatrannfq) [inline src/Repository/AddressRepository.php:17]: Extend Base Repository

File: src/Repository/AddressRepository.php [hunk 2]
```diff
@@ -0,0 +1,66 @@
...
+
+//    /**
+//     * @return Address[] Returns an array of Address objects
+//     */
+//    public function findByExampleField($value): array
+//    {
+//        return $this->createQueryBuilder('a')
+//            ->andWhere('a.exampleField = :val')
+//            ->setParameter('val', $value)
+//            ->orderBy('a.id', 'ASC')
```
Comment (khoatrannfq) [inline src/Repository/AddressRepository.php:50]: Remove unused code

File: src/Repository/AddressRepository.php (hunk 1 above)
Comment (tringuyennfq) [reply src/Repository/AddressRepository.php:17]: okay bro i'll get it fix right now

File: src/Repository/AddressRepository.php (hunk 2 above)
Comment (tringuyennfq) [reply src/Repository/AddressRepository.php:50]: okay bro

**Review by khoatrannfq — APPROVED** [review]
Ok, that's good, you can merge to develop

**Pull request merged**